layout-management-webapps/src/main/resources/locale/portlet/layoutManagement/SiteNavigationPortlet_en.properties

```diff
@@ -7,7 +7,7 @@ siteNavigation.label.deleteSuccess=Node has been deleted.
 siteNavigation.deleteCanceled=Node deletion canceled
 siteNavigation.label.contact.administrator=Please contact your administrator
 siteNavigation.label.manageAccess=Permissions
-siteNavigation.manageAccessDrawer.title=Permissions
+siteNavigation.managePermissionsDrawer.title=Permissions
```

Comment:
siteNavigation.manageAccessDrawer.title should be removed from all languages also